MapboxCoreNavigation/RouteProgress.swift

```diff
@@ -118,6 +118,20 @@ open class RouteProgress: NSObject {
      */
     public var currentLegProgress: RouteLegProgress!
     
+    /**
+     Tupel containing a `CongestionLevel` and a corresponding `TimeInterval` representing the expected travel time for this segment.
+     */
+    public typealias TimedCongestionLevel = (CongestionLevel, TimeInterval)
+    
+    /**
+     If the route contains both `segmentCongestionLevels` and `expectedSegmentTravelTimes`, this array will contain an array of arrays of tuples of `segmentCongestionLevels` and `expectedSegmentTravelTimes`.
```

Comment:
> this array will contain an array of arrays of tuples of `segmentCongestionLevels` and `expectedSegmentTravelTimes`

This isn’t very helpful; Quick Help could’ve told the developer that. On the other hand, it would be helpful to phrase the comment like:

> …this property is set to a deeply nested array of `TimeCongestionLevel`s per segment per step per leg.